config/docker/server/startBroker.sh.mustache

```diff
@@ -1,19 +1,92 @@
 #!/bin/bash
```

Comment:
Why combined all the scripts?

Reply:
I've merged together the recovery script and the start script for each service. I think it gives better and conrtol visibility. The service's run will stay the. Broker and Server scripts are pretty much the same, I'm thinking about merging them together into one.